client/packages/host/getLocalPlugins.js

```diff
@@ -0,0 +1,43 @@
+// This file is used to get a list of plugins to load locally for hot reloading during dev
+// Script expects the file structure to be:
+// packages/plugins/{repository or plugin name}/frontend/latest
+
+const fs = require('fs');
```

Comment:
The reason for this file, is mainly for versioning up the track, when wanted to load an older version of the plugin to test with older version of omSup